cli/cmd/root.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+*/
+package cmd
+
+import (
+	"context"
+	"os"
+
+	"github.com/rilldata/rill/cli/cmd/info"
+	"github.com/rilldata/rill/cli/cmd/version"
+	"github.com/spf13/cobra"
+	// "github.com/spf13/viper"
+
+	"github.com/rilldata/rill/cli/cmd/dropsource"
+	"github.com/rilldata/rill/cli/cmd/example"
+	"github.com/rilldata/rill/cli/cmd/importsource"
+	"github.com/rilldata/rill/cli/cmd/initrill"
+	"github.com/rilldata/rill/cli/cmd/start"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "rill <command>",
+	Short: "Radically simple metrics dashboards",
+	Long:  `Rill makes it easy to create and consume metrics by combining a SQL-based data modeler, real-time database, and metrics dashboard into a single product—a simple alternative to complex BI stacks`,
+	// Uncomment the following line if your bare application
+	// has an action associated with it:
+	// Run: func(cmd *cobra.Command, args []string) { },
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute(ctx context.Context) {
+	ver := "1.0.0"
+	// err := rootCmd.Execute()
+	err := runCmd(ctx, ver)
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func runCmd(ctx context.Context, ver string) error {
+
+	v := version.Format(ver)
+	rootCmd.SetVersionTemplate(v)
+	rootCmd.Version = v
+
+	rootCmd.AddCommand(info.InfoCmd())
+	rootCmd.AddCommand(start.StartCmd())
+	rootCmd.AddCommand(initrill.InitCmd())
+	rootCmd.AddCommand(dropsource.DropSourceCmd())
+	rootCmd.AddCommand(importsource.ImportSourceCmd())
+	rootCmd.AddCommand(example.InitExampleCmd())
+	rootCmd.AddCommand(version.VersionCmd(ver))
```

Comment:
I see you're adding commands here, but there's also a commented-out line for adding the cmd in each cmd's `init` function. You should decide on an approach and remove the other

Reply:
It was default created using cobra-cli, Will add commands in root.go always, as we need, adding them in the init() of each command requires root.go to be accessed and we have created func for each command.
I'll remove the commented thing, Will just keep some basic comment and init() func so that we can use it if needed, (just for our reference) will remove later if not needed